### PR TITLE
New version: Newx.OneMark 0.2.16

### PR DESCRIPTION
**Pull request by risshun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59505)

### CODE REVIEW

File: manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Neux.OneMark
+PackageVersion: 0.2.16
```
Comment (ItzLevvie) [inline manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.installer.yaml:5]:
```suggestion
PackageVersion: 0.2.19
```

File: manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Neux.OneMark
+PackageVersion: 0.2.16
```
Comment (ItzLevvie) [inline manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.2.19
```

File: manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: Neux.OneMark
+PackageVersion: 0.2.16
```
Comment (ItzLevvie) [inline manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 0.2.19
```

File: manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Neux.OneMark
+PackageVersion: 0.2.16
```
Comment (ItzLevvie) [inline manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.yaml:5]:
```suggestion
PackageVersion: 0.2.19
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.installer.yaml (hunk 1 above)
Comment (risshun) [reply manifests/n/Neux/OneMark/0.2.16/Neux.OneMark.installer.yaml:5]: why?😣

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/59505#discussion_r865957752

**Pull request closed without merging**